nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/tag/fn/Functions.java

```diff
@@ -197,8 +197,18 @@ public static String formatDateUsingBasicFormatter(Date date) {
         return formatDate(date, basicDateFormatter());
     }
 
+    public static String formatDateUsingFullDateAndTimeFormatter(Date date) {
```

Comment:
add since annotation please

Reply:
we usually backport the since version from master without bothering to update it to the maintenance branch version (unless it's important for some other reason)

Reply:
(done with #1616)